src/partners_check.c

```diff
@@ -39,14 +39,55 @@
 /*----------------------------------------------------------------------------*/
 /*                             External functions                             */
 /*----------------------------------------------------------------------------*/
+static void parse_partner_id(char *partnerId, char ***partnersList, int *retCount)
```

Comment:
We should avoid triple pointers (`***`).  See the pattern for avoiding this here:
https://stackoverflow.com/questions/21488544/triple-pointers-in-c-is-it-a-matter-of-style

Either a linked list or other data structure will be more descriptive.